### ISSUE TITLE
Test Peek window opens with wrong results selected

### ISSUE THREAD

**Issue by DanTup** [microsoft/vscode#142000]
I ran some tests which failed, and the test peek window opened like this:

<img width="1285" alt="Screenshot 2022-02-02 at 13 45 49" src="https://user-images.githubusercontent.com/1078012/152165809-2f41f2d2-de01-481e-9ad4-e5db51c5eaa8.png">

The test run selected on the right is the one at the bottom of the list, which is the _oldest_ test run. The test run that just finished is at the top of the list.

I would expect the most recent run to be selected.

**Comment by connor4312** [comment]
The tests are inserted in the list in the order in which they were started. I noticed that the error says it timed out after 15 minutes -- are you sure that the test wasn't a 15 minute old run that just finished? Or, in simpler terms, were you able to get this to reproduce consistently? 🙂

**Comment by DanTup** [comment]
@connor4312 it wasn't obvious from the screenshot, but the date I took that screenshot was 2/2 and the result at the top of the list had 2/2 as a date. The results you see in the screenshot are from the previous day.

As far as I can tell, new results are being added to the top of the list, but the selection is always the bottom of the list.

Not on that machine atm, but I can retry again tomorrow (although I'm fairly sure it's always been doing this).

**Comment by DanTup** [comment]
/verified

Looks good in Insiders, thanks!
